requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 pytest==3.2.2
 pytest-xdist==1.20.1
+flake8==3.5.0
```

Comment:
The flake8 dependency is in the `py_ecc[lint]` extra, we should be using that instead.
In fact, requirements-dev and any references to it should be removed entirely.